lib/StatusTypes.ts

```diff
@@ -0,0 +1,34 @@
+export interface CheckStatusRequestBody {
+  birthDate: string
+  esrf: string
+  givenName: string
+  surname: string
+}
+
+export interface CheckStatusReponse {
+  status: string
+}
+
+export interface PassportStatusesSearchResult {
+  _embedded: {
+    passportStatuses: PassportStatusResult[]
+  }
+}
+
+export interface PassportStatusResult {
+  status: string
+}
+
+/**
+ * Map passport status object returned by the API to the client
+ * filter external API fields
+ * @param param0 Passport status object returned by the API
+ * @returns Frontend check status object
+ */
+export function MapToCheckStatusReponse({
```

Comment:
Function name should be in camelCase.

Reply:
We don't have to follow every global convention. Infact most places suggest a team internally decide and agree on the standards they want to follow; to my knowledge we don't have one.

Reply:
It's a javascript convention in general. I'm always trying to follow the language naming convention. To my knowledge, in react you use PascalCase for component names other than that it's camelCase.

Reply:
I feel it's important to follow the industry conventions of the language you are working with. We can have a discussion outside a PR.